eth-types/src/error.rs

```diff
@@ -27,26 +21,6 @@ pub enum Error {
     /// Error when an EvmWord is too big to be converted into a
     /// `MemoryAddress`.
     WordToMemAddr,
-    /// Error while generating a trace.
-    TracingError,
-    /// JSON-RPC related error.
-    JSONRpcError(ProviderError),
-    /// OpcodeId is not a call type.
-    OpcodeIdNotCallType,
-    /// Account not found in the StateDB
-    AccountNotFound(Address),
-    /// Storage key not found in the StateDB
-    StorageKeyNotFound(Address, Word),
```

Comment:
where's  `AccountNotFound`? though we're not using it but we should not remove it. We should also put it in bus-mapping error